### ISSUE TITLE
PTabsItem in PTabs has blue focus outline on Safari

### ISSUE THREAD

**Issue by SetupCoding** [porsche-design-system/porsche-design-system#1698]
Description  
---

![image](https://user-images.githubusercontent.com/46531415/146149690-afa30601-9533-4179-87dd-ebd9c3334ec9.png)

Reported by Julius Adamek https://porschedev.slack.com/archives/CT7AVHTTQ/p1639555510288400

To Reproduce
---
1. Use Safari
1. Go to https://designsystem.porsche.com/latest/components/tabs/examples#basic-example
1. Press Tab multiple times
1. See error

Expected behavior
---
Show focus in all browsers based on the standard definition as 1px border black around the content area including padding (no onclick):
![image](https://user-images.githubusercontent.com/42934679/148980099-f694775e-835c-46aa-a2ab-0a95ada38766.png)

**Accessibility hint**
We may re-think our current implementation where the focused tabpanel has no visual focus indicator. But by following best practices, the tabpanel should get a focus indicator (https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html).

Screenshots
---

Test Environment
---
Platform/Browser/Version: [e.g. iOS Safari 11, macOS Chrome 81]
